### ISSUE TITLE
How to sample  validation set from  training set? 

### ISSUE THREAD

**Issue by swg209** [VisualComputingInstitute/triplet-reid#76]
Dear author,  I read your  wonderful  paper "In Defense of the Triplet Loss for Person Re-Identification",  and I find in the paper that  you sampled  validation set from  training set, so I want to know more detail about how to make it.  Thx

**Comment by Pandoro** [comment]
We never really intended the dataset to be anything people compare results on, especially since we explicitly tune hyperparameters on that validation set. As the papers states, we randomly selected 150 persons from the training set as our validation set. Since not all persons are equally well represented in the training set, we retried sampling those 150 persons until we found a train/validation split that had roughly the same ratio of images as there are persons in the train and validation sets respectively.

That is really all there is no it. I don't know if and where we stored the exact split so we will not be able to provide it, but as stated above, it does not really matter, you can (and should) just create your own similar validation set.
